openerp/addons/base/res/res_partner.py

```diff
@@ -387,6 +393,14 @@ def onchange_state(self, state_id):
             return {'value': {'country_id': state.country_id.id}}
         return {'value': {}}
 
+    @api.onchange('image')
+    def onchange_image(self):
```

Comment:
I'm still not convinced it works correctly, it's weird to try to set the image when the image is changed